gen/compile_proto.jl

```diff
@@ -21,63 +21,47 @@ pbpath =dirname(dirname(PosixPath(pathof(ProtoBuf))))/p"gen"
 cur_path = cwd()
 TBL_root = dirname(cur_path)
 
-src_dir = cur_path/"proto"
-out_dir   = cur_path/"protojl"
+# src_dir = cur_path/"proto"
+src_dir = PosixPath(".")/"proto"
+out_dir = cur_path/"protojl"
 
 ## Clean the output directory
 rm(out_dir, force=true, recursive=true)
 
 ## First module
 function process_module(cur_module::AbstractString; base_module::AbstractString=cur_module, input_path=cur_module)
-    # Include search paths
-    includes   = [src_dir, src_dir/base_module]
 
     # Output folder
-    module_out_dir    = out_dir/cur_module
+    module_out_dir = out_dir/cur_module
 
     # Input files
-    infiles = glob("*.proto", src_dir/input_path)
+    infiles = split.(string.(glob("*.proto", src_dir/input_path)), '/') .|> (a -> a[3:end]) .|> a -> joinpath(a...)
```

Comment:
For readability can we avoid using `|>` ?

This inparticular I think is the same as:
```suggestion
    infiles = glob("proto/$(input_path)/*.proto", src_dir/input_path)), '/')
```

Reply:
In all honesty, this code should probably avoid the use of `PosixPath` altogether, which I used here in the first place because it was already in use and I tried to change as little as possible.
As it is now, it's hard for me to say if this change will work or not. It worked when run, but fails in the REPL. In the interest of my sanity lets do this in a separate PR (which I'll be happy to help with).
Since this whole gen code only needs to run once, lets leave it for now :pray: